### PR TITLE
friday warmup news & location

### PR DESCRIPTION
**Pull request by robertkowalski**

### CODE REVIEW

File: contents/location/index.md [hunk 1]
```diff
@@ -10,3 +13,16 @@ template: simple.jade
 <a href="https://maps.google.com/maps?q=Geomatikum,+Bundesstra%C3%9Fe+55,+Hamburg,+Germany&hl=en&ie=UTF8&ll=53.570288,9.968719&spn=0.036341,0.070295&sll=53.570288,9.968719&sspn=0.073341,0.155354&hq=Geomatikum,+Bundesstra%C3%9Fe+55,+Hamburg,+Germany&t=m&z=14" target="_blank">
     <img height="300" src="http://maps.googleapis.com/maps/api/staticmap?center=53.570288,9.968719&amp;zoom=15&amp;size=635x300&amp;sensor=false&amp;markers=color:0x92137F%7C53.570288,9.968719&amp;visual_refresh=1" width="635">
 </a>
+<br/>
+
+### Friday Warmup Party
+
+**Important:** please RSVP at
+[http://www.meetup.com/hamburg-js/events/151538672/](http://www.meetup.com/hamburg-js/events/151538672/) so the bar owner knows how much staff
+is needed and nobody has to wait for their drinks.
```
Comment (ro-ka) [inline contents/location/index.md:22]: Make it a one-liner.

File: contents/news/friday-warmup/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+We will have a small pre-party with free drinks together with the awesome
+ folks of the [JavaScript Usergroup Hamburg (HH.js)](http://www.meetup.com/hamburg-js/)
+
+It is a 15min walk from U/S Sternschanze.
+
+**Important:** please RSVP at
+[http://www.meetup.com/hamburg-js/events/151538672/](http://www.meetup.com/hamburg-js/events/151538672/) so the bar owner knows how much staff
+is needed and nobody has to wait for their drinks.
```
Comment (ro-ka) [inline contents/news/friday-warmup/index.md:14]: Make it a one-liner.

**Comment by ro-ka** [comment]
Besides the two notes looks good. :)

**Comment by robertkowalski** [comment]
merged this morning with the suggested changes

**Pull request closed without merging**